app/views/public_body/show.html.erb

```diff
@@ -32,22 +37,17 @@
     </p>
   <% end %>
 
-  <% if @public_body.has_notes? || @public_body.eir_only? || @public_body.special_not_requestable_reason? %>
+  <% if @public_body.has_notes? ||
+        @public_body.special_not_requestable_reason? %>
     <div id="stepwise_make_request" class="stepwise_make_request">
       <% if @public_body.has_notes? %>
         <p class="authority__header__notes">
           <%= @public_body.notes_as_html.html_safe %>
         </p>
       <% end %>
 
-      <% if @public_body.is_requestable? %>
-        <% if @public_body.eir_only? %>
-          <p class="authority__header__notes">
-            <%= _('You can only request information about the environment ' \
-                     'from this authority.')%>
-          </p>
-        <% end %>
-      <% elsif @public_body.special_not_requestable_reason? %>
+      <% if @public_body.is_requestable? &&
+            @public_body.special_not_requestable_reason? %>
```

Comment:
This logic looks incorrect.

It can't be `is_requestable?` (`… !defunct? && !not_apply?`) _and_ have `special_not_requestable_reason?` (`defunct? || not_apply?`).

e.g. if I tag an authority with `eir_only, defunct`, I get the EIR message; not the defunct message.

**Expected:**

![screen shot 2018-09-26 at 14 47 34](https://user-images.githubusercontent.com/282788/46084257-2767b600-c19b-11e8-9e5f-f574c040401e.png)

**Actual:**

![screen shot 2018-09-26 at 14 47 09](https://user-images.githubusercontent.com/282788/46084268-2e8ec400-c19b-11e8-8e4e-bb560d529938.png)

Reply:
fixed in 56aec6572

<img width="512" alt="screen shot 2018-09-28 at 12 23 59" src="https://user-images.githubusercontent.com/27760/46206130-f31b0380-c31a-11e8-9c74-9d687b3d346d.png">